src/main/java/com/g4s8/mime/test/HmMimeHasParam.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Kirill (g4s8.public@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.mime.test;
+
+import com.g4s8.mime.MimeType;
+import java.io.IOException;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * MIME param matcher.
+ *
+ * @author Viciouss (xviciouss@gmail.com)
+ * @version $Id$
+ * @since 1.0
+ */
+public final class HmMimeHasParam extends TypeSafeMatcher<MimeType> {
+
+    /**
+     * The parameter name to check.
+     */
+    private final String name;
+
+    /**
+     * Expected matcher.
+     */
+    private final Matcher<String> expected;
+
+    /**
+     * Ctor.
+     * @param name The name of the parameter to be matched.
+     * @param expected The actual value that should be in this parameter.
+     */
+    public HmMimeHasParam(final String name, final String expected) {
+        super(MimeType.class);
```

Comment:
Call primary constructor here please: `this(name, CorMatcher.equalTo(expected))`